hack/unwanted-dependencies.json

```diff
@@ -85,6 +84,7 @@
       ],
       "cloud.google.com/go/compute": [
         "cloud.google.com/go/compute/metadata",
+        "go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc",
```

Comment:
otelgrpc depends on that via `google.golang.org/grpc` (v1.55.0 as of this PR update).  Assuming grpc removes this dependency, it will be removed from otelgrpc in the subsequent release.